README.md

```diff
@@ -44,9 +44,9 @@ To open an RFC, follow these steps:
   * Fill out the RFC template and open a PR.
   * Rename the file to correspond to the GitHub pull request number and update the "RFC PR" field in the file.
 
-The Fellowship will decide, via an on-chain voting mechanism including members III-Dan or above, when to approve and merge RFCs. It does so by issuing an on-chain remark with the body `RFC_APPROVE(xxxx, h)` from the `Fellows` origin on the Polkadot Collectives blockchain, where xxxx is the number of the RFC and h is the blake2-256 hash of the raw proposal text. Once this remark has been made, the PR can be merged. This on-chain process is designed to be resilient to where the RFCs are hosted and in what format, so it can be migrated away from GitHub in the future. The fellowship should not approve more than one RFC with the same number.
+The Fellowship will decide, via an on-chain voting mechanism including members III-Dan or above, when to approve and merge RFCs. It does so by issuing an on-chain remark with the body `RFC_APPROVE(xxxx,h)` from the `Fellows` origin on the Polkadot Collectives blockchain, where xxxx is the number of the RFC and h is the blake2-256 hash of the raw proposal text. Once this remark has been made, the PR can be merged. This on-chain process is designed to be resilient to where the RFCs are hosted and in what format, so it can be migrated away from GitHub in the future. The fellowship should not approve more than one RFC with the same number.
 
-The Fellowship may also decide to reject an RFC by issuing a remark with the text `RFC_REJECT(xxxx, h)`. This is a formality to provide clarity on when PRs (or their analogue on non-GitHub platforms) may be closed. PRs may be closed by their author, as well. PRs may be closed when sufficiently stale, as well - after a period of 1 year without acceptance.
+The Fellowship may also decide to reject an RFC by issuing a remark with the text `RFC_REJECT(xxxx,h)`. This is a formality to provide clarity on when PRs (or their analogue on non-GitHub platforms) may be closed. PRs may be closed by their author, as well. PRs may be closed when sufficiently stale, as well - after a period of 1 year without acceptance.
```

Comment:
Seems better to clarify that the amount of whitespace is irrelevant

Reply:
Having a fixed format makes makes it easier for automation. Otherwise when whitespaces are irrelevant it would be possible to submit anything that matches this regex `\w*RFC_REJECT\w*(\w*xxxx\w*,\w*h\w*)\w*`.  
Unless we specify that a singular whitespace char between the `,` and the hash is irrelevant.

Reply:
Yeah by not allow whitespace can simplify all the related tooling. We should use a tool to generate the remark body anyway so there isn't much reason to accept whitespace.